### DISCUSSION TITLE
bms password change code

### DISCUSSION THREAD

**Discussion by riker65** [syssi/esphome-jk-bms#302] [Q&A]
Hi,

where to change the bms password in the code?

can this be editable via the code? 

Thanks

**Comment by syssi** [comment]
Please have a look at this issue: https://github.com/syssi/esphome-jk-bms/issues/139

TL&DR: You can read/write the BMS without any password.

**Comment by riker65** [comment]
ok thanks, so you just need mac and anyone can write?

**Reply by syssi** [reply]
Correct but the BMS supports a single simultaneous connection. If there is an active connection already a second connection cannot be established.
